app/scripts/collections/forms.coffee

```diff
@@ -205,9 +205,8 @@ define [
     ############################################################################
 
     getFetchAllFieldDBFormsURL: ->
-      url = globals.applicationSettings.get 'baseDBURL'
-      pouchname = globals.applicationSettings.get 'activeFieldDBCorpus'
-      "#{url}/#{pouchname}/_design/pages/_view/datums_chronological"
+      url = globals.applicationSettings.fielddbApplication.corpus.url
+      "#{url}/_design/pages/_view/datums_chronological"
 
```

Comment:
@jrwdunham  yes, a fielddb corpus knows its (current) url (which includes its pouchname), it should be able to take any url, eg if the couchdb is running in a virtual directory, or if its not a couchdb at all but responds to get and post and put requests

```javascript
 url: {
    get: function() {
      if (this.connection && this.connection.corpusUrl) {
        return this.connection.corpusUrl;
      } else {
        if (this.dbname) {
          return this.BASE_DB_URL + "/" + this.dbname;
        }
      }
      this.warn("Using an unlikely url, as if this app was running in a website where the databse is.");
    },

```